code/modules/modular_computers/file_system/programs/generic/records.dm

```diff
@@ -25,7 +25,7 @@
 	var/default_screen = "general"
 	var/record_prefix = ""
 	var/typechoices = list(
-		"physical_status" = list("Active", "*Deceased*", "*SSD*", "Physically Unfit", "Disabled"),
+		"physical_status" = list("Active", "*Deceased*", "*SSD*", "*Missing*", "Physically Unfit", "Disabled"),
```

Comment:
Why does it have `*` around it if it doesn't make it bold? What is the point of it?

Reply:
distinguishment. other entries have it to.